README.md

```diff
@@ -1,41 +1,109 @@
+- \<myservicename\>-service.ts, defines the behavior of the service
+
+To get a basic understanding for developing services, feel free to look into our existing services in the repository. 
+
+## Benutzen von bereits vorhandenen Services
+
+### Herunterladen von Services
+Um einen bereits vorhandenen Beispiel Service zu benutzen, müssen Sie diesen zuersteinmal herunterladen. Dies können Sie unter dem Reiter [releases](https://github.com/roXtra/services/releases) dieses GitHub Repositories tun.  
+
+![Image 1](https://github.com/roXtra/services/blob/Change-README/resources/images/services_releases.PNG "Image 1")
+
+Laden Sie sich die services.zip Ihrer installierten roXtra Version herunter. Diese zip enthält alle Services die zurzeit für diese Version veröffentlicht sind. 
+
+![Image 2](https://github.com/roXtra/services/blob/Change-README/resources/images/services_zip.PNG "Image 2")
+
+### Installieren eines Services
+Nachdem Sie die services.zip Ihrer roXtra Version heruntergeladen haben extrahieren Sie den Service, den Sie installieren möchten, in das Roxtra/eformulare/node_modules/@eformservice Verzeichnis auf Ihrem roXtra Applikationsserver.
+Ggf. muss der Ordner @eformservice von Ihnen erstellt werden.  
+
+![Image 3](https://github.com/roXtra/services/blob/master/resources/images/service_folder.PNG "Image 3")
+
+Um Ihren neuen Service zu verwenden starten Sie Ihren roXtraEFormulare Windows Dienst neu.
+
+## Entwickeln von Services
+
+### Voraussetzungen
+- [Node.js](https://nodejs.org/de/download/) 
+
+### Erste Schritte
+- Klonen Sie das Repository  
+- Öffnen Sie es in VSCode oder in Ihrer bevorzugten Entwicklungsumgebung  
+- Installieren Sie alle Abhängigkeiten mit dem folgenden Konsolenbefehl  
+  ```javascript
+  npm install
+  ````
+- Führen Sie folgenden Befehl aus um alle vorhandenen Services zu bauen
+  ```javascript
+  npm run buildbundle
+  ````
+
+Wenn Sie diese Schritte befolgt haben, wurde eine services.zip mit allen Services im Stammverzeichnis erstellt. Jetzt sind Sie bereit Ihre eigenen Services zu entwickeln.
+
+### Schreiben Sie Ihren eigenen Service
+Services sind in der Programmiersprache TypeScript geschrieben.  
+Ein Service benötigt verschiedene Dateien:  
+- package.json, um die Abhängigkeiten zu definieren
+- tsconfig.json, um Compiler Optionen zu definieren
+- .eslintrc.json, um Linter Optionen zu definieren
+
+Die folgenden Dateien können für Ihren Service angepasst werden:  
+- service.json, definiert die grundlegende Struktur Ihres Services
+- \<myservicename\>-config.tsx, enthält React Code, der das Aussehen Ihres Services bestimmt
+- \<myservicename\>-service.ts, bestimmt das Verhalten Ihres Services
 
-To get a basic understanding for developing services, feel free to look into our example services in the repository. 
+Um ein grundlegendes Verständnis der Service Entwicklung zu erlangen, können Sie sich die bereits vorhandenen Services in diesem Repository anschauen.
```

Comment:
Für ein grundlegendes Verständnis der Service Entwicklung, können Sie gerne einen Blick in die bereits vorhandenen Services in diesem Repository werfen.